driver-core/src/main/com/mongodb/connection/ProtocolHelper.java

```diff
@@ -260,12 +260,11 @@ static void sendCommandStartedEvent(final RequestMessage message, final String d
     }
 
     static void sendCommandSucceededEvent(final RequestMessage message, final String commandName, final BsonDocument response,
-                                          final ConnectionDescription connectionDescription, final long startTimeNanos,
+                                          final ConnectionDescription connectionDescription, final long elapsedTimeNanos,
```

Comment:
Moved responsibility for elapsed time calculation to the caller so that the same value can be used for event generation and event logging.  This required changed to all callers, which are the bulk of the changed files in this PR.

Reply:
Ack